doc/2/api/controllers/security/search-users-by-credentials/index.md

```diff
@@ -0,0 +1,113 @@
+---
+code: true
+type: page
+title: searchUsersByCredentials
+---
+
+# searchUsersByCredentials
+
+Searches users credentials for the specified authentification strategy.
+
+<SinceBadge version="auto-version"/>
+
+---
+
+## Query Syntax
+
+### HTTP
+
+```http
+URL: http://kuzzle:7512/credentials/<strategy>/users/_search
```

Comment:
This URL seems weird to me, I'd rather look for something more similar to the classic user-search API endpoint, like `http://kuzzle:7512/users/_search`. Searching users is what this all is about, isn't it? 

But you probably got to this for some good reason: can anybody give me some context?

Reply:
Ok makes a lot more sense